### PR TITLE
Mjs spy repository

### PR DESCRIPTION
**Pull request by fimoefive**

### CODE REVIEW

File: TomMRiddle-SpyDuh/Controllers/SpiesController.cs [hunk 1]
```diff
@@ -29,7 +29,13 @@ public void GetSpyByID(Guid spyID)
       _spyRepo.GetByID(spyID);
     }
```
Comment (fimoefive) [inline TomMRiddle-SpyDuh/Controllers/SpiesController.cs:31]: getSpyName method

File: TomMRiddle-SpyDuh/DataAccessLayer/SpyRepository.cs [hunk 2]
```diff
@@ -1,93 +1,100 @@
...
+        {
+            var temp = _spies.FirstOrDefault(spy => spy.SpyID == spyID);
+            return temp;
 
-    internal IEnumerable<Spy> GetSpiesBySkill(string skill)
-    {
-      return _spies.Where(Spy => Spy.LSTSkills.Contains(skill));
-    }
+            //return _spies.FirstOrDefault(spy => spy.SpyID == spyID);
+        }
```
Comment (fimoefive) [inline TomMRiddle-SpyDuh/DataAccessLayer/SpyRepository.cs:41]: Get by Name Method in the SpyRepository.cs class

File: TomMRiddle-SpyDuh/Models/Spy.cs [hunk 3]
```diff
@@ -5,6 +5,8 @@ namespace TomMRiddle_SpyDuh.Models
 {
   public class Spy
   {
+
```
Comment (fimoefive) [inline TomMRiddle-SpyDuh/Models/Spy.cs:8]: added spy Name property Model

**Review by caseywalker — APPROVED** [review]
Looks better

**Review by JuanDavila1101 — APPROVED** [review]

**Pull request merged**